src/app/rpmostree-dbus-helpers.c

```diff
@@ -1042,7 +1043,22 @@ rpmostree_sort_pkgs_strv (const char *const* pkgs,
   g_variant_builder_init (&builder, G_VARIANT_TYPE ("ah"));
   for (const char *const* pkg = pkgs; pkg && *pkg; pkg++)
     {
-      if (!g_str_has_suffix (*pkg, ".rpm"))
+      if (g_str_has_prefix (*pkg, "http://") ||
+          g_str_has_prefix (*pkg, "https://"))
+        {
+          g_print ("Downloading '%s'... ", *pkg);
+          glnx_autofd int fd = rpmostree_rs_download_to_fd (*pkg, error);
+          if (fd < 0)
+            return FALSE;
```

Comment:
Should have a  `g_print ("\n");` here.

Reply:
Not if the download fails.  (We have this problem in a lot of other places actually)